pennylane/qnn/__init__.py

```diff
@@ -13,5 +13,6 @@
 # limitations under the License.
 """This module contains classes and functions for constructing quantum neural networks from QNodes."""
 
+from .cost import SquaredErrorLoss
```

Comment:
```suggestion
import pennylane.qnn.cost
```
It might be better to have the cost functions accessible directly from their module rather than crowding `qnn` itself.
E.g. so it is just `qml.qnn.cost.SquaredErrorLoss`.